prelude/prelude.purs

```diff
@@ -70,6 +78,20 @@ module Prelude
   (>>>) :: forall a b c d. (Semigroupoid a) => a b c -> a c d -> a b d
   (>>>) f g = g <<< f
 
+  -- | Categories consist of objects and composable morphisms between them, and as such are
+  -- | semigroupoids, but unlike semigroupoids must have an identity element.
+  -- |
+  -- | Categories should obey the following rules.
+  -- |
+  -- | Association (as semigroupoids)
```

Comment:
I would probably leave the `Semigroupoid` laws out since they are implied by the superclass relationship. It's not so bad here, but for something like `Monad`, listing all the superclass laws could get time-consuming, and this is a good time to establish a default.